web/experimental/new_embed.css

```diff
@@ -102,6 +102,19 @@ body {
   cursor: pointer;
 }
 
+.link-button {
```

Comment:
Any reason you didn't just use an AnchorElement for "show solution?"

Reply:
good idea, fixed